dependent-usage-analyzer/checkout-dependents.sh

```diff
@@ -0,0 +1,28 @@
+# checkout open edx frontend repos
+mkdir .projects
+(
+  cd .projects &&
+  git clone git@github.com:edx/prospectus.git --depth 1
```

Comment:
It might be worth it to put these in alphabetical order so you know you have them all? (I have a keyboard shortcut on vscode for alphabetizing, but I don't remember how I got it. 😛 )

Reply:
👍